operator/config/samples/simulations.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox-echo
+  labels:
+    loggingplumber.isala.me/test: simulations
+spec:
+  containers:
+    - name: busybox
+      image: busybox
+      command: ["sh", "-c", "x=1; while [ $x -gt -1 ]; do echo \"{'count': '$(( x++ ))', 'date': '$(date)'}\" && sleep 5; done"]
+---
+apiVersion: logging.banzaicloud.io/v1beta1
+kind: Flow
+metadata:
+  name: busybox-echo
+  labels:
+    loggingplumber.isala.me/test: simulations
+spec:
+  localOutputRefs:
+    - busybox-echo
+  match:
+    - select:
+        labels:
+          loggingplumber.isala.me/test: invalid
+    - select:
+        labels:
+          loggingplumber.isala.me/test: simulations
```

Comment:
An interesting thing I noticed was all the match statements uses an `OR` to evaluate them. which means if we have a flow with an invalid match first then a flow with a valid one next, the first flow will fail but the second flow will pass even though it has one invalid and one valid. I Hope this isn't an issue.

Reply:
This is fine but it means we cannot short circuit the match logic like we can with filters.  We will need  to fully test all match statements.

Reply:
yes, that's what I was worried about too